site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -351,35 +391,86 @@ When HTTP POST Bindings are used, this is the default verification key used if a
 +
 This field is required when [field]#Require signature# is enabled.
 
+image::core-concepts/application-saml-authentication-response.png[Application SAML authentication response settings,width=1200,role=bottom-cropped top-cropped]
+
+==== Authentication Response
+
+[api]
 [field]#Signing key# [optional]#Optional#::
 The signing key used to sign the SAML request. When this value is not selected the default selection will cause FusionAuth to generate a new key pair and assign it to this configuration.
 
-[field]#XML signature canonicalization method# [optional]#Optional# [default]#defaults to `Exclusive with comments`#::
+[field]#Signature canonicalization method# [optional]#Optional# [default]#defaults to `Exclusive with comments`#::
 The XML signature canonicalization method. If you are unsure which method to select, leave the default and begin testing, or contact your service provider for configuration assistance.
 
-[field]#XML signature location# [optional]#Optional# [default]#defaults to `Assertion`# [since]#Available since 1.21.0#::
+[field]#Signature location# [optional]#Optional# [default]#defaults to `Assertion`# [since]#Available since 1.21.0#::
 The location of the XML signature in the SAML response.
 
-[field]#Response populate lambda# [optional]#Optional#::
+[field]#Populate lambda# [optional]#Optional#::
 The lambda used to add additional values from the user and registration to the SAML response.
 
+image::core-concepts/application-saml-logout-request.png[Application SAML logout request settings,width=1200,role=top-cropped bottom-cropped]
+
+==== Logout Request
+
+[api]
+[field]#Require signature# [optional]#Optional# [since]#Available since 1.25.0#::
+When enabled the SP will be required to sign the Logout request. All unsigned Logout requests will be rejected.
```

Comment:
This might be silly, but what does "SP" stand for here?  Is it defined someplace else in the page/doc?  I only comment because I figure if I don't know our users might also be confused here.

Reply:
Ah, it is 'SP' or service provider (basically, the application which is delegating authentication decisions to the identity provider, which is, in this case, FusionAuth and more particularly this particular fusionauth application). I can spell it out.

Reply:
Added some more details, thanks!